cmd/sentry/sentry/sentry_grpc_server.go

```diff
@@ -736,25 +736,27 @@ func (ss *GrpcServer) SendMessageToRandomPeers(ctx context.Context, req *proto_s
 		return reply, fmt.Errorf("sendMessageToRandomPeers not implemented for message Id: %s", req.Data.Id)
 	}
 
-	amount := uint64(0)
+	peerInfos := make([]*PeerInfo, 0, 32) // 32 gives capacity for 1024 peers, well beyond default
```

Comment:
1. what means `32 gives capacity for 1024 peers`?
2. I think it's fine - if you will do `rangePeers` to calc amount of peers - and know how much need allocate
3. what will happen if some peer is closed after you added it to `peerInfos` array?

Reply:
Aaah ops, this is remnints from a previous implementation that only stored the `log2(total_peers)`.
Since it's just an array of pointers, I think we should either make it grow naturally or default it to `req.MaxPeers`. Calculating it first seems unnecessary, especially since it causes needless locks.

As for what happens if a peer closes after added to `peerInfos`, from reading the code it it uses `sync.Map` internally and `sync.Map.Range()` does not appear to hold a lock when calling `f()` on each iteration, so there should be no difference from the way it is in this PR and the way it was before in terms of lifetime.

ref: https://cs.opensource.google/go/go/+/refs/tags/go1.19.3:src/sync/map.go;l=349